fishjam/server_notifications.proto

```diff
@@ -153,6 +153,28 @@ message ServerMessage {
     Track track = 4;
   }
 
+  // Notification sent when streamer successfully connects
```

Comment:
More general question, should we send this event on connection, or maybe it would be better to send it when the media starts to flow?